File: js/calendario.js
class Calendario {
    constructor(id = null, usuario = null, fecha = null, turno = null) {
        this._id = id;
        this._usuario = usuario;
        this._fecha = fecha;
        this._turno = turno;
    }
    
    get id() {
        return this._id;
    }
    
    set id(id) {
        this._id = id;
    }
    
    get usuario() {
        return this._usuario;
    }
    
    set usuario(usuario) {
        this._usuario = usuario;
    }
    
    get fecha() {
        return this._fecha;
    }
    
    set fecha(fecha) {
        this._fecha = fecha;
    }
    
    get turno() {
        return this._turno;
    }
    
    set turno(turno) {
        this._turno = turno;
    }
}

File: App_Code/Calendario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;
using System.Xml.Serialization;

namespace es.itsduero.iecscyl
{
    [DataContract]
    public class Calendario
    {
        [XmlElement ("CalendarioID")]
        public int ID { get; set; }
        [XmlElement("CalendarioUsuario")]
        public Usuario usuario { get; set; }
        [XmlElement("CalendarioFecha")]
        public DateTime fecha { get; set; }
        [XmlElement("CalendarioTurno")]
        public short turno { get; set; } //0 descanso, 1 mañanas, 2 tardes, 3 noches
        public bool haveTicket { get; set; }
    }
}